lib/archethic/mining/pending_transaction_validation.ex

```diff
@@ -181,19 +179,13 @@ defmodule Archethic.Mining.PendingTransactionValidation do
   end
 
   @spec validate_ownerships(Transaction.t()) :: :ok | {:error, any()}
-  defp validate_ownerships(%Transaction{data: %TransactionData{ownerships: ownerships}}) do
-    nb_ownerships = length(ownerships)
+  defp validate_ownerships(%Transaction{data: %TransactionData{ownerships: []}}), do: :ok
 
-    # defstruct recipients: [], ledger: %Ledger{}`, code: "", ownerships: [], content: ""
-    if nb_ownerships == 0 do
-      # we might not have any ownerships at all
-      :ok
-    else
-      do_validate_ownerships(ownerships)
-    end
+  defp validate_ownerships(%Transaction{data: %TransactionData{ownerships: ownerships}}) do
+    do_validate_ownerships(ownerships)
```

Comment:
I don't think there is need for this function, you could merge it inside the `validate_ownerships`